_studio/hevce_hw/Android.mk

```diff
@@ -0,0 +1,86 @@
+LOCAL_PATH:= $(call my-dir)
+
+MFX_LOCAL_SRC_FILES := $(addprefix h265/src/, \
+    export.cpp \
+    mfx_h265_encode_hw.cpp \
+    mfx_h265_encode_hw_brc.cpp \
+    mfx_h265_encode_hw_bs.cpp \
+    mfx_h265_encode_hw_ddi.cpp \
+    mfx_h265_encode_hw_par.cpp \
+    mfx_h265_encode_hw_utils.cpp \
+    mfx_h265_encode_vaapi.cpp)
+
+MFX_LOCAL_C_INCLUDES := \
+    $(LOCAL_PATH)/h265/include \
+    $(MFX_HOME)/_studio/shared/umc/codec/brc/include
+
+MFX_LOCAL_STATIC_LIBRARIES := \
+    libmfx_trace_hw \
+    libmfx_lib_merged_hw \
+    libumc_core_merged_hw \
+    libsafec
+
+MFX_LOCAL_LDFLAGS := \
+    $(MFX_LDFLAGS) \
+    -Wl,--version-script=$(LOCAL_PATH)/libmfx_h265e_plugin.map \
+    -Wl,--no-warn-shared-textrel
```

Comment:
Why you need this?! Looks like something is very wrong in the build for this target.

Reply:
It was added by @dmitryermilov with the following message: "Fix for "shared library text segment is not shareable" warrning treated as error."
I checked the build without this line, and the build was completed without such error.

Reply:
If you don't see build errors, I suggest to remove.